code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -598,8 +598,12 @@
 		delay = 1 SECONDS
 		playsound(loc, "alien_doorpry", 25, TRUE)
 	else
```

Comment:
Lost the sound-effect via playsound(loc, "alien_doorpry", 25, TRUE)

Could probably be a switch statement

Reply:
just forgot to readd after deleting